build.gradle

```diff
@@ -59,10 +61,10 @@ publishing {
             artifact sourceJar
             pom.withXml {
                 def root = asNode()
-                root.appendNode('name', 'Microsoft Graph SDK for Java')
-                root.appendNode('url', 'https://github.com/microsoftgraph/msgraph-sdk-java')
+                root.appendNode('name', 'Microsoft Graph Auth SDK for Java')
+                root.appendNode('url', 'https://github.com/microsoftgraph/msgraph-sdk-java-auth')
                 root.children().last() + pomConfig
-                def pomFile = file("${project.buildDir}/libs/microsoft-graph.pom")
+                def pomFile = file("${project.buildDir}/libs/microsoft-graph-auth.pom")
```

Comment:
Can you confirm that the pom file is built to this location in the build artifact?

Reply:
It isnt, however the core library is set this way but also does not contain a .pom file in the libs folder either.

Reply:
Hm. It is generated here https://github.com/microsoftgraph/msgraph-sdk-java-auth/blob/dev/build.gradle#L290. Let's give it try as is.